packages/compass-aggregations/src/components/settings/settings.jsx

```diff
@@ -40,10 +39,6 @@ class Settings extends PureComponent {
     this.props.setSettingsSampleSize(parseInt(evt.currentTarget.value, 10));
   }
 
-  onMaxTimeoutChanged(evt) {
```

Comment:
This was unused? 👍

Reply:
Yeah, this was moved out of settings in the toolbar redesign, but some state and related ui code was not cleaned up completely after we removed the old toolbar code